src/bp/core/module-loader.ts

```diff
@@ -139,7 +140,7 @@ export class ModuleLoader {
     // Module loaded successfully, we will process its regular lifecycle
     if (isModuleLoaded) {
       const api = await createForModule(moduleName)
-      await (entryPoint.onServerReady && entryPoint.onServerReady(api))
+      entryPoint.onServerReady && entryPoint.onServerReady(api)
```

Comment:
The typings are incorrect for the entry point, they definitely should be awaited (all of them). 

Otherwise you risk having a long running operation in onServerStarted that isn't completed when onServerReady is being called. Every modules uses async/await in their index.ts files

Reply:
Oh! Thanks for hinting. I removed them because of IDE complaint :)

The methods signature has been changed to return `Promise` instead.